portfolio/src/main/webapp/script.js

```diff
@@ -22,7 +22,83 @@ function addRandomGreeting() {
   // Pick a random greeting.
   const greeting = greetings[Math.floor(Math.random() * greetings.length)];
 
-  // Add it to the page.
+
+        {
   const greetingContainer = document.getElementById('greeting-container');
   greetingContainer.innerText = greeting;
 }
+}
+
+function testKey() {
+    var inputVal = document.getElementById("myInput").value;
+    if(inputVal == "I am Justice I am God" || inputVal == "i am justice i am God" || inputVal == "I am justice I am kira" || inputVal == "I am justice, I am God")
+        {
+            window.alert("You got it right! My favorite TV show is Deathnote!");
+        }
+    else
+        {
+            window.alert("Try again, you almost got it!");
+        }
+}
+/*
+var barelem = document.getElementById("barPercent");
+var totalwidth = elem.style.width;
+window.onload = function pageLoad() {
+    width = 1;
+    elem.style.width = width + '%';
+    elem.innerHTML = width * 1 + '%';
+}
+function testKey() {
+    var inputVal = document.getElementById("myInput").value;
+    var width = 1;
+    var id = setInterval(frame(),10);
+    function frame() {
+        if(inputVal == "I am Justice I am God" || inputVal == "i am justice i am God" || inputVal == "I am justice I am kira" || inputVal == "I am justice, I am God")
+        {
+            if(width>= totalwidth) {
+                clearInterval(id);
+            }
+            else
+            {
+                width++;
+                elem.style.width = width + '%';
+                elem.innerHTML = width * 1 + '%';
+            }
+        }
+    }
+}
+
+//Javascript wasn't working for what I wanted it to so had to scrap original idea as I ran out of time
+*/
+
+async function getServletUsingAsyncAwait() {
+  const response = await fetch('/data');
+  const quote = await response.text();
+  document.getElementById('servlet-container').innerText = quote;
+}
+
+function getMessage() {
```

Comment:
This doesn't seem to be used anywhere?

Reply:
Deleted, will be updated in next Commit/PR